src/commands/sentinel-config.json

```diff
@@ -77,6 +77,12 @@
                                     "const": "unknown"
                                 }
                             ]
+                        },
+                        "tls-cert-file": {
+                            "type": "string"
+                        },
+                        "tls-key-file": {
+                            "type": "string"
                         }
```

Comment:
please update also history section in this file.

Reply:
Since I have to put a version number in the history, what would be the correct number?

Reply:
I think 7.2.0 will be ok.